lib/command/run.rb

```diff
@@ -197,13 +195,22 @@ def create_runner_workload # rubocop:disable Metrics/MethodLength
       end
     end
 
-    def update_runner_workload # rubocop:disable Metrics/MethodLength
-      step("Updating runner workload '#{runner_workload}'") do # rubocop:disable Metrics/BlockLength
-        @expected_deployed_version = cp.cron_workload_deployed_version(runner_workload)
-        should_update = false
+    def update_runner_workload # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity
+      should_update = false
+      spec = nil
 
+      step("Checking if runner workload '#{runner_workload}' needs to be updated") do # rubocop:disable Metrics/BlockLength
+        _, original_container_spec = base_workload_specs(original_workload)
```

Comment:
Separated the previous step in 2 steps, because I feel like printing `Checking if runner workload ... needs to be updated` is more useful than printing `Updating runner workload ...` every time.

So now `Updating runner workload ...` is only printed if the runner workload is actually being updated.